meta/main.yml

```diff
@@ -12,4 +12,5 @@ galaxy_info:
   #   versions:
   #   - 2013.09
   galaxy_tags: [amazon, centos6, nginx, php, laravel]
-dependencies: []
+dependencies:
+  - role: retr0h.logrotate
```

Comment:
```
dependencies:
  - { role: retr0h.logrotate, when: laravel_rotate_nginx_log }
```
Is this what you want? If yes, how we set the variable when downloading the role?